### DISCUSSION TITLE
Spiff up Portfolio

### DISCUSSION THREAD

**Discussion by BioNCphilic** [earthlab-education/Earth-Analytics-AY24#557] [Show and tell]
Hey @ZimmZone - I love how your portfolio page is divided and I was looking at your code. Did you use an editor? I wanted to follow your style, but I've not really used CSS before (only html). Just curious how you put your name & info in a different section of your page using the index.md format.

**Comment by ZimmZone** [comment]
@BioNCphilic Hi!  I am glad you liked it. I totally used an editor that did the formatting for me. If you look in my config file (or the attached screenshot) you can see how I set it up. It is the Minimal theme.
<img width="1046" alt="image" src="https://github.com/user-attachments/assets/4b9bb023-eab3-44ad-925f-8d65a4fe051d">

**Reply by ZimmZone** [reply]
Good luck and let me know if you have any questions.

**Comment by nquarder** [comment]
Nice work @ZimmZone! I'm using the same theme for my portfolio page as well (minimal). @BioNCphilic there is a section in the textbook that has some instruction about how to create the `_config.yml` file and working with different Jekyll themes. 

https://cu-esiil-edu.github.io/esiil-learning-portal/foundations/pages/03-git-github/03-github-portfolio/04-jekyll.html

**Comment by BioNCphilic** [comment]
Thanks! 
I got it to work: [check it out](https://bioncphilic.github.io/)
